nbgrader/docs/source/contributor_guide/releasing.rst

```diff
@@ -5,12 +5,15 @@ Releasing a new version
     :local:
     :depth: 1
 
+This repository uses `jupyter_releaser <https://jupyter-releaser.readthedocs.io/en/latest/>`_.
+
 Prerequisites
 -------------
 
 - All PRs should be tagged with the appropriate milestone. Several of the
   releasing scripts depend on this being done.
 - You will need a GitHub API key: https://github.com/settings/tokens
+- Install packaging requirement `tbump <https://github.com/dmerejkowsky/tbump>`_ : ``pip install tbump``
```

Comment:
I guess this should already be installed when you do the dev install, so perhaps it would be more general to say something like "make sure you have done a development install" and link to the relevant docs?